api/test/e2e/route_with_management fileds_test.go

```diff
@@ -0,0 +1,282 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package e2e
+
+import (
+	"io/ioutil"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/tidwall/gjson"
+)
+
+func TestRoute_with_name_desc(t *testing.T) {
+	tests := []HttpTestCase{
+		{
+			caseDesc: "config route with name and desc (r1)",
```

Comment:
Why is the first letter lowercase of caseDesc?

Reply:
Now the case does not require the first letter of caseDesc has to be capitalized, all existing use cases are in lower case, I think it can be regarded as "standard".

Reply:
> @idbeta I think it can be regarded as "standard".

that is totally wrong. I created a related issue yesterday: https://github.com/apache/apisix-dashboard/issues/912
we'll fix this later. 

`caseDesc` is not a good field name

Reply:
ok, I see, we can fix it in `2.2`, what do you think? @nic-chen